### PR TITLE
Rani

### PR DESCRIPTION
**Pull request by Rany615**
Resolved previous pull request comments
added back authentication which was temporarily disabled before

### CODE REVIEW

File: src/runningMethods/recovery.js [hunk 1]
```diff
@@ -13,11 +13,12 @@ module.exports = async () => {
         fn.dataSources.aka,
         fn.dataSources.es,
         fn.dataSources.ads, 
-        fn.dataSources.adNN, 
+        /*fn.dataSources.adNN,
```
Comment (NoamYes) [inline src/runningMethods/recovery.js:16]: remove commeneted dataSOurces

File: scripts/checkForDuplicates/checkForDuplicates.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        let isDuplicate = await isDuplicatePerson(akaRecord,sendLog)
+        if (isDuplicate){
+            haveDuplicates.push(akaRecord)
+        }
+    };
+    var file = fs.createWriteStream('DuplicatePersons.txt');
+    // file.on('error', function(err) { /* error handling */ });
+    file.write(JSON.stringify(haveDuplicates))
+    file.end();
+    })();
```
Comment (NoamYes) [inline scripts/checkForDuplicates/checkForDuplicates.js:24]: add empty row at the end of each file

File: scripts/checkForDuplicates/checkForDuplicates.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    let { sendLog, dataObj } = await preRun(fn.runnigTypes.recoveryRun, [fn.dataSources.aka]);
+    let akaData = dataObj[fn.dataSources.aka] ? dataObj[fn.dataSources.aka].data : [];
+    let haveDuplicates = []
+    for (akaRecord of akaData){
+        let isDuplicate = await isDuplicatePerson(akaRecord,sendLog)
+        if (isDuplicate){
+            haveDuplicates.push(akaRecord)
+        }
+    };
+    var file = fs.createWriteStream('DuplicatePersons.txt');
```
Comment (NoamYes) [inline scripts/checkForDuplicates/checkForDuplicates.js:20]: const instead of var

File: src/runningMethods/recovery.js (hunk 1 above)
Comment (Rany615) [reply src/runningMethods/recovery.js:16]: remove or uncomment?

Comment (NoamYes) [reply src/runningMethods/recovery.js:16]: we want them of course, keep them just remove the comments

File: scripts/checkForDuplicates/checkForDuplicates.js [hunk 4]
```diff
@@ -17,8 +17,9 @@ fs = require('fs');
             haveDuplicates.push(akaRecord)
         }
     };
-    var file = fs.createWriteStream('DuplicatePersons.txt');
+    const file = fs.createWriteStream('DuplicatePersons.txt');
     // file.on('error', function(err) { /* error handling */ });
     file.write(JSON.stringify(haveDuplicates))
+    file.write('\r\n');
     file.end();
     })();
```
Comment (NoamYes) [inline scripts/checkForDuplicates/checkForDuplicates.js:25]: empty row at each of source file, not of the data file of duplicates
Can be indetified by the icon 
image

**Pull request closed without merging**